docs/UserGuide.md

```diff
@@ -198,6 +198,7 @@ Deletes the specified person(s) from the address book.
 
 Format: `delete [INDEX | INDEX-INDEX]…`
 
+* At least one of the index (range) must be provided.
```

Comment:
maybe it'll be clearer if its "At least one index or index range must be provided."?